lib/src/delegates/asset_picker_builder_delegate.dart

```diff
@@ -11,6 +11,7 @@ import 'package:flutter/material.dart';
 import 'package:flutter/gestures.dart';
 import 'package:flutter/scheduler.dart';
 import 'package:flutter/services.dart';
+import 'package:wechat_assets_picker/src/widget/scale_text.dart';
```

Comment:
We don't import files in src as package. Also after the suggestion this line should be resorted to the correct line number.

```suggestion
import '../widget/scale_text.dart';
```

Reply:
Fixed all local package imports to relative imports